tutorials/mongoose/enums.md

```diff
@@ -0,0 +1,39 @@
+Mongoose `String` and `Number` types had access to the `enum` validator.
+The `enum` validator is an array that will check if the value given is an item in the array.
+If it is not, it will throw an error.
```

Comment:
If the value is not in the array, Mongoose will throw a `ValidationError` when you try to `save()`.

In general, avoid using "it" twice in the same sentence to refer to different concepts.